src/localsocketcontroller.cpp

```diff
@@ -22,11 +23,9 @@
 #include "models/server.h"
 #include "settingsholder.h"
 
-// How many times do we try to reconnect.
-constexpr int MAX_CONNECTION_RETRY = 10;
-
 // How long do we wait between one try and the next one.
-constexpr int CONNECTION_RETRY_TIMER_MSEC = 500;
+constexpr int CONNECTION_RETRY_INITIAL_MSEC = 500;
+constexpr int CONNECTION_RETRY_MAXIMUM_MSEC = 16000;
```

Comment:
Can you expand on the comment on line 26 to explain why there are 2 values? Do we retry every 500ms, up until 16000ms?

Reply:
Done!